exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/__init__.py

```diff
@@ -0,0 +1,78 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This library allows to export tracing data to an OTLP collector.
+
+Usage
```

Comment:
Any reason why we have the docs in this folder? Would it make sense to move to `trace_exporter/__init__.py`? Do we even need an `__init__.py` here?

Reply:
related issue https://github.com/open-telemetry/opentelemetry-python/issues/1678. Initially, all the Exporter code was in this init but to follow the same pattern as in grpc `opentelemetry.exporter.otlp.proto.grpc.trace_exporter` moved the code to trace_exporter.